tests/trainer/test_dataloaders.py

```diff
@@ -1665,3 +1665,47 @@ def on_validation_batch_start(self, batch, batch_idx: int, dataloader_idx: int)
     assert model.on_train_batch_start_called
     assert model.on_val_dataloader_called
     assert model.on_val_batch_start_called
+
+
+@RunIf(min_gpus=2, special=True)
+@pytest.mark.parametrize([
+    "total_train_samples",
+    "train_batch_size",
+    "total_val_samples",
+    "val_batch_size",
+    "val_check_interval",
+], [
+    pytest.param(8, 4, 2, 1, 0.2),
+    pytest.param(8, 4, 2, 1, 0.5),
+])
+def test_progress_bar_max_val_check_interval(
```

Comment:
I think this test would fit better in the train loop tests file

Reply:
Actually, it would be better in progress `test_progress_bar.py`